### PR TITLE
[doc] support ngram_search function

### PR DESCRIPTION
**Pull request by Mryange**
https://github.com/apache/doris/pull/38226

### CODE REVIEW

**Review by luzhijing — COMMENTED** [review]
Please add it in branch-3.0

**Review by luzhijing — APPROVED** [review]
LGTM

File: docs/sql-manual/sql-functions/string-functions/ngram_search.md
Comment (KassieZ) [inline docs/sql-manual/sql-functions/string-functions/ngram_search.md:1]: please capitalize title

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Review by KassieZ — APPROVED** [review]

**Pull request merged**